dotcom-rendering/src/web/components/NavList.tsx

```diff
@@ -52,29 +47,42 @@ const liStyles = (containerOverrides?: ContainerOverrides) => css`
 	}
 `;
 
-export const NavList = ({ trails, containerPalette }: Props) => {
+const textColumns = css`
+	${until.tablet} {
+		column-count: 1;
+	}
+	${between.tablet.and.desktop} {
+		column-count: 3;
+	}
+	column-count: 4;
```

Comment:
We tend to work mobile-first, so we should probably switch this around and go for:
```css
column-count: 1;
${from.tablet} {
  column-count: 3;
}
${from.desktop} {
  column-count: 4;
}
```